### ISSUE TITLE
Using wasd control, physics-body would be stuck while moving.

### ISSUE THREAD

**Issue by daoshengmu** [ngokevin/aframe-physics-components#11]
I notice that if we use `'velocity' in diffData`, we can't update the velocity continuously. I am not quite sure whether cannon.js will reset the velocity internally. If it would, we should keep feeding the velocity while pressing wasd keys.

https://github.com/ngokevin/aframe-physics-components/blob/master/src/index.js#L101

**Comment by ngokevin** [comment]
Yeah, try https://github.com/donmccurdy/aframe-extras which has more physics work at this point.
